### PR TITLE
feat: add manypkg for mono-repo management

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

This is similar to app-kit and internally. It will help us to manage so many packages and eventually we can replace lerna.

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
-  "extends": ["config:base", ":preserveSemverRanges", "schedule:weekly"],
+  "extends": [
+    "config:base",
+    "pinOnlyDevDependencies",
```
Comment (emmenko) [inline .github/renovate.json:4]: no need for this

**Review by emmenko — COMMENTED** [review]
👍

File: .github/renovate.json (hunk 1 above)
Comment (tdeekens) [reply .github/renovate.json:4]: Just wanted to keep things in sync. I don't mind. I can remove it.

Comment (emmenko) [reply .github/renovate.json]: Exactly, our OSS repos don't use this anymore 😉
Thanks!

**Comment by tdeekens** [comment]
Taking the 👍🏻 as an approval. Also as this type of change has been approved across various other repositories.

**Pull request merged**